core/web/src/main/java/edu/uci/ics/texera/web/resource/QueryPlanResource.java

```diff
@@ -1,5 +1,7 @@
 package edu.uci.ics.texera.web.resource;
 
+import edu.uci.ics.texera.api.engine.MultipleSinkPlan;
```

Comment:
move these imports close to similar imports.

Reply:
Fixed